docs/arch/adr-31.md

```diff
@@ -0,0 +1,87 @@
+# ADR 31: RotatingRectangle Tool and TemporalRotatingRectangle Tool
+
+## Status
+
+In Review
+
+## Context
+
+The Solar Jets project requires a TemporalRotatingRectangle drawing tool for volunteers to capture the width, height, angle and time properties of a solar jet. At the time of this writing, the `front-end-monorepo` has a standard Rectangle drawing tool which gives the width and height properties. Based on the standard Rectangle drawing tool, we want to create a RotatingRectangle drawing tool and a TemporalRotatingRectangle drawing tool.
+
+The RotatingRectangle drawing tool is a common tool used in the Panoptes code based and eventually needs to be written in the `front-end-monorepo`. This is a good opportunity to complete this task.
+
+The idea is to create two new rectangle drawing tools based off the standard Rectangle drawing tool. once complete, the `front-end-monorepo` will have the following rectangle drawing tools:
+
+**Rectangle**
+
+```json
+{
+  "details": array,
+  "frame": number,
+  "height": number,
+  "toolIndex": number,
+  "toolType": string,
+  "width": number,
+  "x_center": number,
+  "Y_center": number
+}
+```
+
+**RotatingRectangle**
+
+\*everything the `Rectangle` tool has plus this addition:
+
+```json
+{
+  "angle": number
+}
+```
+
+**NOTE:** angle is Degrees (-180, 180)
```

Comment:
This should also note that positive angles rotate from `+x` to `+y` (a right-handed coordinate system).  If using SVG with the origin in the upper left, and `+y` downward positive angles rotate clockwise.  See a full discussion of this in https://github.com/zooniverse/front-end-monorepo/issues/500 

The rotation should also be about the center of the rectangle.